Release/tests/functional/utils/datetime.cpp

```diff
@@ -332,6 +331,8 @@ SUITE(datetime)
         {
             auto dt = utility::datetime::from_string(str, utility::datetime::RFC_1123);
             VERIFY_ARE_EQUAL(0, dt.to_interval());
+            auto dt_me = utility::datetime::from_string_maximum_error(str, utility::datetime::RFC_1123);
+            VERIFY_ARE_EQUAL(utility::datetime::maximum(), dt_me);
```

Comment:
doesn't this test code against itself? That is from_string_maximum_error is based on datetime::maximum()?

Reply:
The contract of `from_string_maximum_error` is "returns `datetime::maximum()` on error", so this is testing that it returned an error.